app/components/workflows/workflow-detail.js

```diff
@@ -2,4 +2,12 @@ import Ember from 'ember';
 
 export default Ember.Component.extend({
   workflow: null,
+  onlyShowCurrent: false,
+  sortedVersions: Ember.computed('onlyShowCurrent', 'workflow.versions', function() {
+    const versions = this.get('workflow.versions').sortBy('version').reverse();
+    if (this.get('onlyShowCurrent')) {
+      return versions.slice(0, 1);
```

Comment:
FYI, Ember adds some convenient, observable properties to [Array](https://www.emberjs.com/api/ember/2.15/classes/Ember.Array/) like [firstObject](https://www.emberjs.com/api/ember/2.15/classes/Ember.Array/properties/firstObject?anchor=firstObject). Handy if like me, you can't remember how to use `slice()` and `splice()`

So you can write this line as versions.get('firstObject'), and sometimes even move to a computed property alias